src/web/services/student.service.ts

```diff
@@ -129,10 +136,42 @@ export class StudentService {
    * Loads list of students from a course in CSV format by calling API.
    */
   loadStudentListAsCsv(queryParams: { courseId: string }): Observable<string> {
-    const paramsMap: Record<string, string> = {
+    return this.httpRequestService.get(ResourceEndpoints.COURSE, {
```

Comment:
Actually you can consider use `CourseService#getCourseAsInstructor` and `StudentService#getStudentsFromCourse`.

Reply:
I wanted to do that at first but thought it might unnecessarily couple the course service with the student service. I'll go change it :P